src/Symfony/Component/ClassLoader/ClassMapGenerator.php

```diff
@@ -134,7 +134,7 @@ private static function findClasses($path)
                     }
 
                     if ($isClassConstant) {
-                        continue;
+                        break;
```

Comment:
I fully understand =) 

Or take in account finally-blocks introduced in 5.5 - if you get an exception inside (even handled) you can have variety of miss-behaviors of code in different PHP versions.